libpoolprotocols/stratum/EthStratumClient.cpp

```diff
@@ -592,6 +604,14 @@ void EthStratumClient::submitSolution(Solution solution) {
 	string nonceHex = toHex(solution.nonce);
 	string json;
 
+	m_responsetimer.cancel();
+
+	if (m_response_pending)
```

Comment:
why this? also m_stale gets overwritten a bit later in the same method

Reply:
No good reason. Leftover from previous attempt. Fixing.